portfolio/src/main/webapp/script.js

```diff
@@ -31,7 +31,8 @@ function change() {
 }
 
 function getComments() {
-  fetch('/data').then(response => response.json()).then((comments) => {
+  let displayValue = document.getElementById("comment-choice").value;
```

Comment:
```javascript
const displayValue
```

Consider doing some client-side validation here - while it can't replace your server-side validation, it would help reduce erroneous inputs sent to the server.